beater/server_test.go

```diff
@@ -612,6 +613,7 @@ func TestServerOutputConfigReload(t *testing.T) {
 
 	runServerArgs := <-runServerCalls
 	assert.Equal(t, "", runServerArgs.Config.Sampling.Tail.ESConfig.Username)
+	assert.Equal(t, "custom", runServerArgs.Namespace)
```

Comment:
At first I was trying to verify that this was "default" and then below it would change, but I was getting the following error trying to update that config:

```
=== RUN   TestServerOutputConfigReload
    server_test.go:625: 
        	Error Trace:	server_test.go:625
        	Error:      	Received unexpected error:
        	            	more than one namespace configured accessing config
        	Test:       	TestServerOutputConfigReload
```

I wasn't sure how to fix it. But, seeing as a user might use a custom namespace when first installing apm-integration, I figured this also tests the code path.

Reply:
Yeah I think this is perfectly reasonable.